test/Main.hs

```diff
@@ -27,17 +28,18 @@ main :: IO ()
 main = do
     selected <- getArgs
     let go current = when (null selected || current `elem` selected)
-    go "Acyclic.AdjacencyMap"   testAcyclicAdjacencyMap
```

Comment:
Please bring back the dots.

Reply:
Ups, I don't know how those disappeared! Everything is fixed now!